### DISCUSSION TITLE
Consolidating Outcomes

### DISCUSSION THREAD

**Discussion by rachaelbradley** [w3c/wcag3#101] [General]
We would like to consolidate the outcomes so that subgroups consider all related outcomes. 

Please review the [proposal for consolidation](https://docs.google.com/document/d/1GL-jyZ7CuYb-fms_jTKEe1YTAUZAH0EXkU3yqbRg5f4/edit#heading=h.8n8glns7lrs4) and add comments or propose an alternative. 

What changes are needed for subgroup work?

**Comment by rachaelbradley** [comment]
Consider breaking "Adjust Color" apart into a level under text contrast, focus visible, and component contrast.

Are we missing an outcome for nontext static content contrast?

**Reply by rachaelbradley** [reply]
Broader question for consideration during the meeting: Should outcomes in user control that are related to other outcomes such as adjust color and text customization be grouped with related outcomes?

I’ve suggested moving a few other outcomes in the document.

**Comment by bruce-usab** [comment]
I have a few other comments in the doc, but the only one I feel strongly about is having seizure triggers in the same grouping as Avoiding Motion and Audio Shifting.

**Reply by bruce-usab** [reply]
From call, I now understand the logic for the grouping (thank you).  My concern is narrowed in that we have hard research for [thresholds defined by safety standards](https://www.w3.org/TR/WCAG22/#bib-iso_9241-391) for Flashing and Strobing.  Is there anything comparable for other vestibular triggers?   Can we add that phrasing (thresholds defined by safety standards) to Avoiding Motion and Audio Shifting?

**Comment by GreggVan** [comment]
agree.  we also hadsound pattern issues that came up in previous WCAG discussion.   We did  a lot of research on those and found that there was no set pattern and they were real but ideopathic.  For example, a particular symphony would consistently cause a seizure but other symphonies that were similar or even by same composer had not effect.     So we should not ever say "that isnt real" but we do need to only create rules for things that are clearly defined and a widespread problem.     For the rest we need to work on prophylactic measures that can recognize a trigger and mask it.   We are getting very close to that capability now.
